### DISCUSSION TITLE
Project Permissions - Can't Access any Project Issues Made Before 4.27 Update

### DISCUSSION THREAD

**Discussion by sean-cannon** [community/community#54363] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

Created a [ticket](https://support.github.com/ticket/personal/0/2130419) last week, but haven't gotten any response. 

Our current problem is that 4/5 of our team members are unable to access any Project Issues that were created pre-4.27.2023, when there was apparently [a release specifically targeting permissions in GH Projects](https://github.blog/changelog/2023-04-27-github-issues-projects-april-27th-update/). Our admin is able to see Issues, but any other team member is unable, even if their role is changed to admin.

We technically started experiencing this late Wednesday afternoon, (4.26), as our admin was adjusting permissions for the team. He changed our roles from individual admins to a team with write permissions. Once that happened, we were unable to see any Issues. What would normally have 50+ Issues only listed a single task (see screenshot below):

<img width="1507" alt="GH Projects Bug - No Issues" src="https://user-images.githubusercontent.com/111802362/235939194-8d4d3485-2993-485c-abb3-a8ecf9f637fd.png">

When we attempt to go to an Issue link, we get a 404:

<img width="1512" alt="GH Projects Bug - 404 on Issue Link" src="https://user-images.githubusercontent.com/111802362/235939186-6e5a4093-3847-4994-90b3-c51a92af9218.png">

This is really halting our project management, as our admin is somewhat removed from day-to-day development enough that it's very inefficient for him to be the only one to show the board on a given day. 

Anyone else experiencing this issue? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by traumverloren** [comment]
:wave: Hi! I'm one of the engineers working on Projects. I would love to debug this and unblock your work here. I'm happy to debug in this discussion or we can move to your [ticket](https://support.github.com/ticket/personal/0/2130419). 

I'm curious if it's related to the move of individuals to a team and if the team lacks permissions on these repositories (since you can't access the issues directly from a url as well). Could you have an admin check what the permission level are set to for the base role & team in the repository settings > collaborators and teams view for the repo in the screenshot above with 404? The repo settings  > collaborators and teams view will look like this:

![Screenshot 2023-05-19 at 12 38 59](https://github.com/community/community/assets/9959680/6d8284e4-ea49-4a43-8f61-68319d427c84)

**Reply by sean-cannon** [reply]
Hi @traumverloren , thanks for following up on this! The issue has since been resolved; turns out it was a misunderstanding on our end.
